Python/ast_unparse.c

```diff
@@ -697,6 +723,32 @@ append_formattedvalue(_PyUnicodeWriter *writer, expr_ty e, bool is_format_spec)
     APPEND_STR_FINISH("}");
 }
 
+static int
+append_ast_constant(_PyUnicodeWriter *writer, PyObject *constant)
+{
+    if (PyTuple_CheckExact(constant)) {
+        int ret;
+        Py_ssize_t i, elem_count;
+
+        elem_count = PySequence_Length(constant);
+        if (elem_count == -1) {
+            return -1;
+        }
+        APPEND_STR("(");
+        for (i = 0; i < elem_count; i++) {
+            APPEND_STR_IF(i > 0, ", ");
+            if ((ret = append_repr(writer, PyTuple_GET_ITEM(constant, i))) != 0) {
```

Comment:
ret can be removed, there is no need to return ret, just return -1 if the function result is negative:

```suggestion
            if (append_repr(writer, PyTuple_GET_ITEM(constant, i)) < 0) {
```